cves/kernel/CVE-2021-3659.yml

```diff
@@ -320,8 +324,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: "There was an Acked-by which I had to look up to figure out what that meant but yes, there
```

Comment:
Remove the mention of looking it up to figure out what Acked-by means. Can just mention that the commit was Acked-by one person and then signed off and committed by another person.